docs/getting_started/singularity.md

```diff
@@ -0,0 +1,109 @@
+# Running Labelmerge with Singularity
+
+## Pre-requisites
+1. Singularity / Apptainer is is installed on your system. For more info, see
+the detailed [Apptainer install instructions](https://apptainer.org/docs/admin/main/installation.html#install-from-pre-built-packages).
+1. The following command-line tools are installed:
+    * wget
+1. Sufficient disk-space (rough estimate)
+    * in your `/tmp` folder (>30GB) to build the container
+    * in your working folder to store the container (~20GB)
+    * for SCATTR outputs (~40GB per subject using default parameters)
+1. Sufficient CPU and memory - the more you have, the faster it will run and 
```

Comment:
Probably don't need 8 cores and 64GB memory for an example Labelmerge

Reply:
I don’t think we need more than 8gb maybe 16g for labelmerge